lib/ohai/plugins/packages.rb

```diff
@@ -31,6 +31,7 @@
   end
 
   collect_data(:linux) do
+    require "date"
```

Comment:
Can you move this into the Arch section since it's not needed for the other platforms